XMonad/Hooks/DynamicLog.hs

```diff
@@ -261,6 +261,5 @@ dynamicLogXinerama = withWindowSet $ io . putStrLn . pprWindowSetXinerama
 xmobarProp :: LayoutClass l Window
            => XConfig l  -- ^ The base config
            -> IO (XConfig (ModifiedLayout AvoidStruts l))
-xmobarProp conf = do
-    xmobarPropConfig <- statusBarProp "xmobar" (pure xmobarPP)
-    withEasySB xmobarPropConfig toggleStrutsKey conf
+xmobarProp conf = return
```

Comment:
I wonder if dropping this would be a good idea as well, so it behaves more like `withEasySB`

Reply:
`xmobarProp` was merged in late December and we're hoping people won't use DynamicLog any more, so it doesn't matter much. We might break a config or two, but then this might be considered a good thing as it will notify people who follow the bleeding edge stuff that there is a bleedinger edger stuff in X.H.StatusBar. :-)

(I'm not really sure why we decided to keep this one addition in DynamicLog, so if anyone knows, they're in a better position to decide whether to change this type or not.)

Reply:
Hmm, good points. Should we move it then? And maybe re-export it?

Reply:
I meant "keep" as in "not remove because it wasn't there at 0.16". It definitely belongs in DynamicLog, it's just a question whether it should exist at all.

Reply:
Oh, I think it might nudge people to checkout `X.H.StatusBar`, so maybe it stays?

Reply:
I think it's nice having it there as a sort of "look what `X.H.StatusBar` can do".  Perhaps this is wishful thinking, but I think it would drive people exploring the docs towards at least checking out these modules